src/Init.h

```diff
@@ -60,6 +60,7 @@ namespace Init {
 	void	FileRead_Mode(Simdat&, const string&);
 	void	FileRead_Mode_Snapshot(Simdat&, const string&);
 	void	FileRead_Mode_Solidification(Simdat&, const string&);
+	void	FileRead_Mode_Stork(Simdat&, const string&);
```

Comment:
Is this definition missing or not needed?

Reply:
@JamieStumpORNL @gknapp1 this declaration seems out of place - that's my only current issue here

Reply:
Should be no longer needed